hw/ip/otbn/dv/uvm/otbn_sim_cfg.hjson

```diff
@@ -23,24 +24,40 @@
 
   // Import additional common sim cfg files.
   // TODO: remove imported cfgs that do not apply.
-  import_cfgs: [// Project wide common sim cfg file
-                "{proj_root}/hw/dv/tools/dvsim/common_sim_cfg.hjson",
+  import_cfgs: [
+      // Project wide common sim cfg file
+      "{proj_root}/hw/dv/tools/dvsim/common_sim_cfg.hjson",
+      // Config file to pull in the correct flags for otbn_memutil
+      "{proj_root}/hw/ip/otbn/dv/memutil/otbn_memutil_sim_opts.hjson"
+  ]
+
+  // Add options needed to compile against otbn_memutil
+  en_build_modes: ["{tool}_otbn_memutil_build_opts"]
 
   // Add additional tops for simulation.
   sim_tops: ["-top {name}_bind"]
 
   // Default iterations for all tests - each test entry can override this.
-  reseed: 50
+  reseed: 1
```

Comment:
is one seed enough? I think we have some rand variables.

Reply:
It's definitely not enough! :-) But it means that I don't have to keep remembering the `--fixed-seed` parameter when doing bring-up. This will definitely get bumped up in the future.

Reply:
I see. it might be easier to review and also remind to update this later if you add a TODO

Reply:
Eh, ok, I'll do that.

I'd note that "loadsa iterations" is a pretty terrible way to ensure we're done testing: that's what functional coverage is for! And, of course, once we have coverage points we'll notice that they aren't getting filled, so I'm not sure that the TODO helps much. Can't hurt, though.